# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/websocket/WebSocketChannel.cpp
# Commit: 64b4a5c112b5
# Full Hash: 64b4a5c112b5bec70574f3d63a751302bbc5460b
# Author: Marian-Vasile Laza <mlaza@mozilla.com>
# Date: 2022-02-12 09:47:43
# Regressor Bug: 1749786
# File Overlap Count: 1
# Description:
#   Bug 1749786 - lint fix r=fix CLOSED TREE
# ==============================================================================

diff -r 8a0a3e09cf26 -r 64b4a5c112b5 netwerk/protocol/websocket/WebSocketChannel.cpp
--- a/netwerk/protocol/websocket/WebSocketChannel.cpp	Fri Feb 11 17:19:56 2022 +0000
+++ b/netwerk/protocol/websocket/WebSocketChannel.cpp	Fri Feb 11 19:41:54 2022 +0200
@@ -432,8 +432,8 @@
     {
       MutexAutoLock lock(aChannel->mMutex);
       MOZ_ASSERT(
-        NS_FAILED(aReason) || aChannel->mScriptCloseCode == CLOSE_GOING_AWAY,
-        "websocket closed while connecting w/o failing?");
+          NS_FAILED(aReason) || aChannel->mScriptCloseCode == CLOSE_GOING_AWAY,
+          "websocket closed while connecting w/o failing?");
     }
 #endif
     Unused << aReason;
@@ -2247,7 +2247,7 @@
 void WebSocketChannel::CleanupConnection() {
   // normally this should be called on socket thread, but it may be called
   // on MainThread
-  
+
   LOG(("WebSocketChannel::CleanupConnection() %p", this));
   // This needs to run on the IOThread so we don't need to lock a bunch of these
   if (!mIOThread->IsOnCurrentThread()) {
@@ -2348,7 +2348,8 @@
     MutexAutoLock lock(mMutex);
     if (mReconnectDelayTimer) {
       mReconnectDelayTimer->Cancel();
-      NS_ReleaseOnMainThread("WebSocketChannel::mMutex", mReconnectDelayTimer.forget());
+      NS_ReleaseOnMainThread("WebSocketChannel::mMutex",
+                             mReconnectDelayTimer.forget());
     }
   }
 
@@ -3262,7 +3263,7 @@
     }
 
     AbortSession(NS_ERROR_NET_TIMEOUT);
-  // mReconnectDelayTimer is only modified on MainThread
+    // mReconnectDelayTimer is only modified on MainThread
   } else if (timer == mReconnectDelayTimer) {
     MOZ_RELEASE_ASSERT(mConnecting == CONNECTING_DELAYED,
                        "woke up from delay w/o being delayed?");